### DISCUSSION TITLE
Proper usage?

### DISCUSSION THREAD

**Discussion by jonpoploskie** [philosowaffle/peloton-to-garmin#199] [Q&A] [answered]
Hello, I apologize if this is covered somewhere and I just missed it… when starting a ride on the peloton should I also start an indoor ride activity on my fenix 6? That’s what I have been doing but then when I run p2g I get 2 workouts. The p2g one has power data and other great stuff, the fenix one has some of the proprietary metrics from garmin. Is there a way to merge them? Otherwise it seems I just end up deleting the garmin one. Or should I just have the peloton connect to hr on the fenix, not start an activity on the watch, and just have the one activity after import. 
thank you!

**Comment by philosowaffle** [comment, marked as answer]
Do you happen to remember what extra metrics are showing up from your watch recording that are not present in the P2G workout? Maybe a screenshot? (just curious)

Generally, I use my watch to broadcast HR to the bike, but I [do not save the watch workout](https://philosowaffle.github.io/peloton-to-garmin/features.html#typical-usage) at the end of the ride (I click delete instead of save on my watch).  Atleast for me, the workout synced from P2G also has the proprietary stuff in it like Training Stress Score and Intensity Factor.  There's also a bit of discussion about P2G and Training Status/Effect [over here](https://github.com/philosowaffle/peloton-to-garmin/issues/114#issuecomment-901113919).


![brave_2021-12-30_15-18-50](https://user-images.githubusercontent.com/7275041/147788799-e8e5440e-1a01-4ec8-b69e-9d0822005541.png)


I believe [SyncMyWorkout](https://syncmyworkout.com/) will attempt to merge the two workouts (watch + peloton) and then delete the original watch workout automagically for you. At this time, I don't feel comfortable implementing something like that in P2G because I think the risk is too high to accidentally mess up or delete the original data.

Another option if you **really** want to merge the two workouts together, but did not want to pay for SyncMyWorkout, is to merge the files manually using [FIT File Tools](https://www.fitfiletools.com/#/combiner#view). I've used this online tool before to merge together treadmill data + watch data, and it worked well for me.  But it is a fully manual process.

**Reply by delta0815** [reply]
Hi all,

Did this decision change over time? I just wonder as I am using SyncMyWorkout and the merge/delete process works pretty well and you get the best from both worlds. So I was hoping that this option might be something that will become available in the future :-)

Thanks for your thoughts on this

Cheers,

Oliver

**Comment by jonpoploskie** [comment]
[edited so screenshots showed up]
Thanks so much for your reply! Here is a screenshot of a Garmin indoor ride
activity, highlighting in yellow the items I don't see in the p2g one below
it. Just to be clear none of these things are all that important, I'm just
kind of curious if all that data can be captured/merged. I think you
already explained basically everything I'm showing below, so probably all
good unless you see something odd? Great suggestion too on not saving the
workout from the watch, I'll give that a go. Still learning about the 10
billion functions on the watch. :-) Thanks!
![image](https://user-images.githubusercontent.com/54714036/147795959-6bb4cd1a-e958-4b87-b4c7-fc14adc04218.png)

Here is the p2g version:
![image](https://user-images.githubusercontent.com/54714036/147795974-13683deb-199a-4df8-b410-25454181c384.png)

**Reply by philosowaffle** [reply]
Thanks for the screenshots! I can confirm those fields you circled are all data provided on the Garmin watch itself and they show up because SyncMyWorkout has merged the watch data with the Peloton data, something P2G does not do at this time.
